docs/topics/windows-vhd.md

```diff
@@ -0,0 +1,53 @@
+# Windows AKS Azure marketplace images
+
+## Motivation
+
+The primary motivation for producing AKS specific Windows VHDs is to improve the reliability and deployment time for the configuration phase of Windows nodes configured by aks-engine. 
+By performing expensive configuration options and pre-downloading installation artifacts as part of the VHD build process we can accomplish both of these goals.
+
+Producing and publishing AKS specific Windows VHDs to the Azure marketplace also allow us to get targeted windows patches to customers sooner. 
+Today the WIndows/Windows Server team only publish new images containing the latest B week patches. Several times recently C week patches contain fixes to issues customers are facing running Kubernetes on Windows nodes. This means customers would either need to wait an additional 3-4 weeks to get these fixes or another mechanism to developed these patches to Windows nodes would need to be developed.
+
+Lastly publishing AKS specific Windows VHDs allows us to perform adequate testing on new patches before allowing customers to customers to upgrade their Windows nodes.
+In the first half of 2019 monthly security fixes broke Kubernetes workloads on Windows a few different occasions forcing the the aks-engine team to start hard-coding the specific versions of the Windows Server marketplace images (instead of targeting latest). 
+This needs to be resolved by collaborating and testing with the Windows team before patches get released publicly regardless (see [Testing Private Fixes](#Testing-Private-Fixes) section below).

```

Comment:
Maybe rephrase? How about something like, "Given the challenging nature of Windows fixes, we will work with the Windows team to include Kubernetes testing before public release of patches. In the mean time the VHD pipeline supports [Testing Private Fixes](#Testing-Private-Fixes)"